src/date-input/date-input.tsx

```diff
@@ -44,6 +44,7 @@ export const DateInput = ({
     const nodeRef = useRef<HTMLDivElement>(null);
     const calendarRef = useRef<InternalCalendarRef>();
     const inputRef = useRef<StandaloneDateInputRef>();
+    const MIN_DATE_INPUT = "1900-01-01";
```

Comment:
Instead of setting it as such, you should set "1900-01-01" over in the deconstruct of the props instead at Line 16